Dockerfile

```diff
@@ -1,40 +1,14 @@
-FROM jrottenberg/ffmpeg:ubuntu
-COPY . /opt/shinobi
+FROM mhart/alpine-node:8
+
 WORKDIR /opt/shinobi
+RUN apk add --update ffmpeg python pkgconfig cairo-dev make g++ jpeg-dev && \
+    rm -rf /var/cache/apk/*
 
```

Comment:
yep, originally i had multiple lines of `apk add` to save the caching
but since its one line i've updated the commit as per your suggestion, thanks!